.github/workflows/build-images.yml

```
@@ -269,6 +266,89 @@ jobs:
         echo -n $(docker inspect --format '{{index .Config.Labels "vertica-version"}}' ${{ steps.nokeys_vertica_image.outputs.value }}) >> $GITHUB_STEP_SUMMARY
         echo "**" >> $GITHUB_STEP_SUMMARY
 
+  build-server-v2:
+    runs-on: ubuntu-latest
+    outputs:
+      image: ${{ steps.v2_vertica_image.outputs.value }}
+    steps:
+
+    - name: set lower case owner name
+      env:
+        OWNER: '${{ github.repository_owner }}'
+      run: |
+        echo "OWNER_LC=${OWNER,,}" >>${GITHUB_ENV}
+
+    - name: Pick the name of the image
+      uses: spilchen/switch-case-action@v2
+      id: v2_vertica_image
+      with:
+        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-nokeys
+        conditionals-with-values: |
+          ${{ inputs.v2_vertica_image != '' }} => ${{ inputs.v2_vertica_image }}
+          ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind-nokeys
```

Lets not use -nokeys as the default image name. Instead, lets use -v2.
```suggestion
        default: ghcr.io/${{ env.OWNER_LC }}/vertica-k8s:${{ github.sha }}-v2
        conditionals-with-values: |
          ${{ inputs.v2_vertica_image != '' }} => ${{ inputs.v2_vertica_image }}
          ${{ github.event_name == 'pull_request' }} => vertica-k8s:kind-v2
```